components/omega/doc/design/OmegaV1GoverningEqns.md

```diff
@@ -0,0 +1,765 @@
+
+$$
+p(x,y,z) = p^{surf}(x,y) + \int_{z}^{z^{surf}} \rho g dz'.
+$$ (continuous-hydrostatic-pressure-final)
+
+Here the $\nabla$ operators are exactly horizontal; we expand terms for tilted layers in the following section. The momentum diffusion terms include Laplacian (del2), biharmonic (del4), and vertical viscosity,
+
+$$
+{\bf D}^u
+= \nu_2 \nabla^2 {\bf u} - \nu_4 \nabla^4 {\bf u}
+ + \frac{\partial }{\partial z} \left( \nu_v \frac{\partial {\bf u}}{\partial z} \right)
+$$ (continuous-h_mom_diff)
+and may also include a Rayleigh drag and eventually parameterizations. Momentum forcing is due to wind stress and bottom drag. Similarly, the tracer diffusion terms include Laplacian (del2), and vertical viscosity,
+
+$$
+D^\varphi =
+ \nabla\cdot\left(\rho \kappa_2 \nabla\varphi \right)
++ \rho \frac{\partial }{\partial z}
+  \left( \kappa_v \frac{\partial \varphi}{\partial z} \right),
+$$ (continuous-v_tr_diff)
+
+and may also include a biharmonic (del4) term and parameterizations such as Redi mixing. Sources and sinks include surface fluxes from the atmosphere and land, and bio-geo-chemical reactions.
+All of the diffusion and forcing terms are written in more detail with the [Discrete Equations](#discrete-equations) below.
+
+## 4. Layered Equations
+
+Omega-1 will be non-Boussinesq, which means that density is not assumed to be constant in any terms. This results in a mass conservation equation, rather than a volume conservation equation. The fully continuous equations were presented in the previous section. Here we derive the layered equations by discretizing in the vertical, while the horizontal remains continuous. We discretize by integrating in the vertical from the lower surface $z=z_k^{bot}(x,y)$ to $z=z_k^{top}(x,y)$ for the layer with index $k$, as described in [Ringler et al. 2013](https://www.sciencedirect.com/science/article/pii/S1463500313000760) Appendix A.2. Equivalently, we can vertically integrate from a deeper pressure surface $p=p_k^{bot}(x,y)$ (higher pressure) to $p=p_k^{top}(x,y)$ where $p$ and $z$ are related by the hydrostatic pressure equation [](#continuous-hydrostatic-pressure).
+
+### Layer Integration
+
+For non-Boussinesq layered equations we begin by defining the mass-thickness of layer $k$ as
+
+$$
+h_k(x,y,t)
+ \equiv \int_{z_k^{bot}}^{z_k^{top}} \rho dz = \frac{1}{g} \int_{p_k^{top}}^{p_k^{bot}} dp.
+$$ (def-h)
+
+The letter $h$ is used because this is the familiar variable for thickness in Boussinesq primitive equations. In the Boussinesq case the thickness equation describes conservation of volume because $h_k^{Bouss}$ is volume normalized by horizontal area, resulting in a height,
+
+$$
+h_k^{Bouss}(x,y,t)
+ \equiv \int_{z_k^{bot}}^{z_k^{top}} dz.
+$$ (def-h-bouss)
+
+In this document we remain with the more general non-Boussinesq case, where $h_k$ is mass per unit area (kg/m$^2$). Since horizontal cell area remains constant, the thickness equation is a statement of conservation of mass.
```

Comment:
```suggestion
In this document we remain with the more general non-Boussinesq case, where $h_k$ is mass per unit area (kg/m$^2$). Since horizontal cell area remains constant in time, the thickness equation is a statement of conservation of mass.
```